extensions/amp-story/1.0/amp-story-store-service.js

```diff
@@ -219,6 +223,7 @@ export const Action = {
   SET_PAGE_SIZE: 'updatePageSize',
   ADD_PANNING_MEDIA_STATE: 'addPanningMediaState',
   SET_VIEWER_CUSTOM_CONTROLS: 'setCustomControls',
+  TOGGLE_SUBSCRIPTION_DIALOG_IS_VISIBLE: 'toggleSubscriptionDialogIsVisible',
```

Comment:
Nit: A-Z

Reply:
Done. Existing lines are not ordered so let me just sort them all.